### PR TITLE
Low-code CDK: Update autogenerated docs

### PR DESCRIPTION
**Pull request by scottleechua**
## What
Fixes issue [#18658](https://github.com/airbytehq/airbyte/issues/18658).

## How
- Add command to run integration tests using Docker
- Replace static documentation URL in template README.md with dynamic URL.
- Update `docs.airbyte.io` to `docs.airbyte.com`

## 🚨 User Impact 🚨
Minimal.

## Pre-merge Checklist

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))

### CODE REVIEW

File: airbyte-integrations/connector-templates/source-configuration-based/README.md.hbs [hunk 1]
```diff
@@ -48,10 +48,14 @@ docker run --rm -v $(pwd)/secrets:/secrets -v $(pwd)/integration_tests:/integrat
...
 #### Acceptance Tests
-Customize `acceptance-test-config.yml` file to configure tests. See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference) for more information.
+Customize `acceptance-test-config.yml` file to configure tests. See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference) for more information.
 If your connector requires to create or destroy resources for use during acceptance tests create fixtures for it and place them inside integration_tests/acceptance.py.
 
-To run your integration tests with docker
+To run your integration tests with docker:
+```
+docker build . -t airbyte/source-exchange-rates-tutorial:dev
+python -m pytest integration_tests -p integration_tests.acceptance
```
Comment (sherifnada) [inline airbyte-integrations/connector-templates/source-configuration-based/README.md.hbs:57]: This runs integration tests using python - this should suggest using acceptance-tests-docker.sh

**Review by sherifnada — COMMENTED** [review]
@scottleechua thanks for the improvement! Made one request for change and we should be good to go.

File: airbyte-integrations/connector-templates/source-configuration-based/README.md.hbs (hunk 1 above)
Comment (scottleechua) [reply airbyte-integrations/connector-templates/source-configuration-based/README.md.hbs]: @sherifnada ah, thanks for that! Updated the command. 👍

File: airbyte-integrations/connector-templates/source-configuration-based/source_{{snakeCase name}}/{{snakeCase name}}.yaml.hbs [hunk 2]
```diff
@@ -35,7 +35,7 @@ check:
     - "customers"
 
 spec:
-  documentation_url: https://docsurl.com
+  documentation_url: https://docs.airbyte.com/integrations/sources/{{dashCase name}}
```
Comment (marcosmarxm) [inline airbyte-integrations/connector-templates/source-configuration-based/source_{{snakeCase name}}/{{snakeCase name}}.yaml.hbs:38]: thank you so much! :D

**Review by marcosmarxm — APPROVED** [review]
Thanks @scottleechua I'll merge tomorrow, I want to validate we don't need to regenerate any files.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**